docs/README.md

```diff
@@ -182,7 +182,9 @@ Options:
   --help            Show this message and exit.
 
 Commands:
-  plot     Install charts with given arguments as listed in yaml file...
+  plot     Install charts with given arguments as listed in yaml file argument
```

Comment:
These end up coming across as:

```
  plot      Install charts with given arguments as listed in yaml file...
  template  Output the template of the chart or charts as they would be...
  version   Takes no arguments, outputs version info
```

Do we need to make them shorter?

Reply:
I don't think so because if you do `reckoner plot --help` you get the full string 
You only get the truncated string with you do `reckoner --help`

I chose to copy/pasta the long string into the docs to make it easier to read them though.

Reply:
Ah, cool. Seems fine then